development/Dockerfile

```diff
@@ -18,6 +18,10 @@ FROM ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER}-py${PYTHON_VER}
 # Runtime argument and environment setup
 ARG NAUTOBOT_ROOT=/opt/nautobot
 
+# Docker needs to specify the build args in each stage
```

Comment:
It looks like this is a duplicate of  lines 10 and 13.

Reply:
That was intentional. I updated the comment and removed the one that wasn't being used

Reply:
I guess I need to pull this down, Github is playing tricks on me.

Reply:
Pulled it down, it looks to be this way due to docker caching.  It might make more sense to move the ARG to just before we need it, to make it more explicit your doing this to prevent using a bad cache. 

```
ARG CI
# The build args must be declared before use to prevent a cache issue
ARG PYTHON_VER
IF ...
```